gemfiles/activerecord_4.2.gemfile

```diff
@@ -7,7 +7,7 @@ gem "activerecord", "~> 4.2.0"
 platforms :ruby do
   gem "mysql2", "< 0.5"
   gem "pg", "~> 0.21"
-  gem "sqlite3"
+  gem "sqlite3", "~> 1.3", "< 1.4"
```

Comment:
I think these gemfiles have been fixed at their source in appraisals.

Could you remove the changes to the gemfiles from this PR and ensure that the tests work for you?